cmd/gitserver/internal/git/gitcli/mergebase.go

```diff
@@ -47,3 +47,44 @@ func (g *gitCLIBackend) MergeBase(ctx context.Context, baseRevspec, headRevspec
 
 	return api.CommitID(bytes.TrimSpace(stdout)), nil
 }
+
+func (g *gitCLIBackend) OctopusMergeBase(ctx context.Context, revspecs ...string) (api.CommitID, error) {
```

Comment:
```suggestion
func (g *gitCLIBackend) OctopusMergeBase(ctx context.Context, commits ...api.CommitID) (api.CommitID, error) {
```

https://git-scm.com/docs/git-merge-base

Arbitrary revspecs like `A..B` don't seem to be allowed based on the docs, just commits.

Reply:
mmm interesting, I went off of mergebase, which is probably wrong then as well 😬

Reply:
Turns out gits documentation lies and we do rely on this taking revspecs - which are indeed valid parameters here.

Reply:
Oh what you are passing are ranges, revspecs are this https://git-scm.com/docs/git-rev-parse.html#_specifying_revisions.
So SHAs, tags, branches, branch at date, ... 

`git merge-base HEAD@{yesterday} 5.3` for example is valid